SonarQube.Common/DefaultSonarPropertyValues.cs

```diff
@@ -0,0 +1,17 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="DefaultSonarPropertyValues.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace SonarQube.Common
+{
+    /// <summary>
+    /// Defines default values for some properties
+    /// </summary>
+    public static class DefaultSonarPropertyValues
```

Comment:
@bgavrilMS IMO in a perfect world only the bootstrapper would have to worry about computing the SonarQube server URL, and the preprocessor would only re-use that computed value